build/images/ovs/Dockerfile

```diff
@@ -39,7 +53,7 @@ COPY charon-logging.conf /tmp
 # final image.
 RUN apt-get update && \
     apt-get install -y --no-install-recommends iptables logrotate libstrongswan-standard-plugins && \
-    (dpkg -i /tmp/ovs-debs/*.deb || apt-get -f -y --no-install-recommends install) && \
+    apt-get -f -y --no-install-recommends install /tmp/ovs-debs/*.deb && \
```

Comment:
I don't remember this change in the original PR and not sure what's the impact. Why is the change required?

Reply:
Ignore this, just found your explanation above.